attributes/default.rb

```diff
@@ -90,6 +90,9 @@
 if platform_family?('smartos')
   default['openssh']['server']['host_key'] = ['/var/ssh/ssh_host_rsa_key', '/var/ssh/ssh_host_dsa_key']
 end
+if platform_family?('rhel') && node['platform_version'].to_i == 6
```

Comment:
Can you collapse this into the if statement above so we can avoid defining the attributes twice